package.json

```diff
@@ -54,5 +54,9 @@
         12
       ]
     }
-  }
+  },
+  "files": [
```

Comment:
a package.json whitelist approach is very dangerous, and tests *should* be published to npm. At the very least, please use `npmignore`.